rllib/tuned_examples/pg/cartpole-crashing-pg.yaml

```diff
@@ -30,4 +35,12 @@ cartpole-crashing-pg:
         evaluation_duration_unit: episodes
         evaluation_parallel_to_training: true
         evaluation_config:
-          explore: false
+            explore: false
+            env_config:
+                config:
+                    # Make eval workers solid.
+                    # This test is to prove that we can learn with crashing env,
```

Comment:
Do you need disable_env_checking even with these numbers that forces the env not to crash?

Reply:
yeah I do. env can crash during the actual checking part of the env_checker, which will then crash the entire test.